<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelDlg</class>
 <widget class="QDialog" name="ChannelDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>滑槽组</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>460</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>星空联盟</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>已用滑槽编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>空闲滑槽编号</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/add.png</normaloff>:/HuacaoManagement/Resources/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnDel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>158</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/del.png</normaloff>:/HuacaoManagement/Resources/del.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QListWidget" name="huacaoUsed">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>100</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="huacaoFree">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>100</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAdd_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>245</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/1197037.png</normaloff>:/HuacaoManagement/Resources/1197037.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="btnDel_1">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>245</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/1197030.png</normaloff>:/HuacaoManagement/Resources/1197030.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>245</y>
       <width>50</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>天合联盟</string>
    </attribute>
    <widget class="QToolButton" name="btnDel2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>158</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/del.png</normaloff>:/HuacaoManagement/Resources/del.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>已用滑槽编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>空闲滑槽编号</string>
     </property>
    </widget>
    <widget class="QListWidget" name="huacaoFree2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>100</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAdd2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/add.png</normaloff>:/HuacaoManagement/Resources/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="huacaoUsed2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>100</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAdd_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>245</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/1197037.png</normaloff>:/HuacaoManagement/Resources/1197037.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="btnDel_2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>245</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/1197030.png</normaloff>:/HuacaoManagement/Resources/1197030.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>245</y>
       <width>50</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HuacaoManagement.qrc"/>
 </resources>
 <connections/>
</ui>
